var/spack/repos/builtin/packages/flex/package.py

```diff
@@ -31,10 +31,10 @@ class Flex(AutotoolsPackage):
     homepage = "https://github.com/westes/flex"
     url = "https://github.com/westes/flex/releases/download/v2.6.1/flex-2.6.1.tar.gz"
 
-    version('2.6.3', 'a5f65570cd9107ec8a8ec88f17b31bb1')
```

Comment:
This is the error, which is why I hacked the flex spec. The error occurs when building something that depends_on("flex")

    ==> Error: Invalid spec: 'flex@2.6.3%gcc@4.8 arch=linux-opensuse42-x86_64 ^bison@3.0.4%gcc@4.8 arch=linux-opensuse42-x86_64 ^bzip2^gettext@0.19.8.1%gcc@4.8+bzip2+curses+git~libunistring+libxml2+tar+xz arch=linux-opensuse42-x86_64 ^help2man@1.47.4%gcc@4.8 arch=linux-opensuse42-x86_64 ^libsigsegv^libxml2^m4@1.4.18%gcc@4.8+sigsegv arch=linux-opensuse42-x86_64 ^mawk^ncurses^pkg-config@0.9.0:^tar^xz^zlib@1.2.11%gcc@4.8+pic+shared arch=linux-opensuse42-x86_64'.
    Package flex requires version :2.6.1, but spec asked for 2.6.3